src/components/Layout/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -83,10 +85,12 @@ const Breadcrumbs = () => {
           return (
             <li className={styles.breadLine} key={item}>
               <Link
-                className={`${styles.breadLink} ${
-                  index === pathnames().length - 1 && styles.currentPath
-                }`}
-                to={`/${item}/${allCategories?.[0]?.id}`}
+                className={`${styles.breadLink} ${index === pathnames().length - 1 && styles.currentPath}`}
+                to={
+                  index === 1
+                    ? `/catalog/${allCategories?.find((category) => category.name === item)?.id}`
+                    : `/${item}`
+                }
```

Comment:
why do you have this statement? `index === 1`

Reply:
this was done to determine whether the current segment is a product category path in the breadcrumb system.